src/components/menu/styles.ts

```diff
@@ -0,0 +1,68 @@
+import { styled } from '@linaria/react'
+import { Text2XS } from '../typography'
+
+export const ElMenuPopover = styled.div`
+  position: absolute;
+  white-space: nowrap;
+  z-index: 98;
+`
```

Comment:
**question:** Can you help me understand why these styles are necessary on the popover?

Reply:
yes, the context is typically this component will open triggered by a button, and they should have a `relative` parent, that is what the `position: absolute and z-index` for the `white-space` is so that the list item text not collapsed
![image](https://github.com/user-attachments/assets/f98e6bef-832a-47ad-8ff2-51133a2d691d)

Reply:
> does the design spec show menu item text wrapping like this?

currently no, I think it because the example only for standard sized text, so long text might be rare

Reply:
Okay, could be worth confirming whether we need text wrapping or not. We can do this as part of the next PR if you'd like, so long as we don't forget to remove this `white-space` styling from the popover once we know how items should/should not handle long label text.

Reply:
yeah agree, I I'll remove the `white-space` in this pr